cores/arduino/CDCSerialClass.cpp

```diff
@@ -73,7 +100,9 @@ void CDCSerialClass::init(const uint32_t dwBaudRate, const uint8_t modeReg)
      * Empty the Rx buffer but don't touch Tx buffer: it is drained by the
      * LMT one way or another */
     _rx_buffer->tail = _rx_buffer->head;
-
+    
+    mailbox_register(6, cdc_mbox_isr);
```

Comment:
From my initial tests using channel 7 for both was not stable. It freezes when we are sending and receiving on both cores at a high rate